N3.js/src/main/scala/org/w3/banana/n3js/io/N3jsTurtleParser.scala

```diff
@@ -0,0 +1,45 @@
+package org.w3.banana
+package n3js
```

Comment:
You mean having the package split in two? It is mostly equivalent except for imports. I believe that the package name matches the directories in sesame/jena.